readme.md

```diff
@@ -298,6 +298,7 @@
 	- [Typing](https://github.com/typeddjango/awesome-python-typing#readme) - Optional static typing for Python.
 	- [MicroPython](https://github.com/mcauser/awesome-micropython#readme) - A lean and efficient implementation of Python 3 for microcontrollers.
 - [Rust](https://github.com/rust-unofficial/awesome-rust#readme)
+ 	- [Pest](https://github.com/pest-parser/awesome-pest#readme) - Parser generator.
```

Comment:
```suggestion
- [Pest](https://github.com/pest-parser/awesome-pest#readme) - Parser generator.
```

Reply:
It's a library / parser generator for Rust, so I think its list level should be under Rust, not the same level as Rust. Is that correct @sindresorhus?